packages/core-components/src/layout/Sidebar/Items.tsx

```diff
@@ -291,57 +290,6 @@ function isButtonItem(
 
 const sidebarSubmenuType = React.createElement(SidebarSubmenu).type;
 
-// TODO(Rugvip): Remove this once NavLink is updated in react-router-dom.
```

Comment:
@Rugvip some eyes on this one

Reply:
Yep this should now be fixed, best to verify though